app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
```

Comment:
Have you used a constraint layout before? For this type of view it might be easier (You don't need to do this in this PR, you can try it out and change it in a separate PR.)

Reply:
I never tried. I'll check!